pygmt/src/surface.py

```diff
@@ -26,14 +26,14 @@ def surface(x=None, y=None, z=None, data=None, **kwargs):
     Surface reads randomly-spaced (x,y,z) triples and produces gridded values
     z(x,y) by solving:
 
-        (1 - T) * L (L (z)) + T * L (z) = 0
+    .. math::    (1 - T)\nabla^2(z)+t\nabla(z) = 0
```

Comment:
```suggestion
    .. math::    (1 - t)\nabla^2(z)+t\nabla(z) = 0
```
GMT Docs at https://docs.generic-mapping-tools.org/6.1/surface.html uses small letter t.